src/install/requirements_common.txt

```diff
@@ -2,34 +2,26 @@
 testresources==2.0.1
 
 # General python dependencies
-appdirs==1.4.4
```

Comment:
There is a corresponding mock in `docsrc/conf.py` that needs to be removed.
Same for `flaky`. I'm not sure if there are others since it is hard to see which dependencies were removed.